Game/MyProject/Game/Game.cpp

```diff
@@ -89,8 +89,8 @@ Game::PostRendererInitialize()
 void
 Game::UpdateObjects( float dt )
 {
-     _scene_graph->updateScene(dt);
-     test->Update(dt);
+    _scene_graph->updateScene(dt);
+    test->Update(dt);
```

Comment:
I think you should call `test->Update(dt)` before `updateScene` because any changes to nodes position will only be visible in the next frame.